apps/posclient-front/styles/globals.css

```
@@ -225,19 +399,31 @@
+}
+
+.receipt-print__ddtd {
+  display: block !important;
+  width: 100%;
+}
+
+.receipt-print__ddtd-value {
+  display: block;
+  width: 100%;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)